### PR TITLE
Persist volume across videos

### PR DESCRIPTION
**Pull request by furstenheim**
The volume preference is preserved during the same session.
#574

### CODE REVIEW

File: renderer/index.js [hunk 1]
```diff
@@ -458,7 +458,8 @@ function setVolume (volume) {
   if (isCasting()) {
     Cast.setVolume(volume)
   } else {
-    state.playing.setVolume = volume
```
Comment (grunjol) [inline renderer/index.js]: I think this is the way to dispatch that you need to change the volume to the <video> tag . 
Also, this is only for the local renderer and not the cast devices

**Comment by furstenheim** [comment]
Which tag do you refer to? As for the cast devices, do you have any idea how can I mock one to do that part as well?

**Comment by grunjol** [comment]
setVolume holds the value to be set in the <video> DOM element. thats why is set and cleared in the UI.

**Comment by furstenheim** [comment]
@grunjol it should be better now. In any case, only for the local renderer.

**Comment by feross** [comment]
Thanks for the PR, @furstenheim!

This works for now, but it would be better if this was a permanent setting and if it was rendered directly into the volume control html as a default starting volume.

Thanks for taking a stab at this!

**Comment by feross** [comment]
Rebased and merged your commits onto master. GitHub is showing this PR as not merged, but it really is.

**Pull request closed without merging**